.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
-xcuserdata
\ No newline at end of file
+xcuserdata
+.idea
```

Comment:
I prefer to only keep Xcode/macOS specific things, otherwise there will be a lot of things here